easybuild/easyconfigs/l/LLVM/LLVM-12.0.1-GCCcore-10.3.0.eb

```
@@ -0,0 +1,43 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'LLVM'
+version = '12.0.1'
+
+homepage = "https://llvm.org/"
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+toolchainopts = {'cstd': 'gnu++11'}
+
+source_urls = ['https://github.com/llvm/llvm-project/releases/download/llvmorg-%(version)s/']
+sources = ['llvm-%(version)s.src.tar.xz']
+checksums = ['7d9a8405f557cefc5a21bf5672af73903b64749d9bc3a50322239f56f34ffddf']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake', '3.20.1'),
+    ('Python', '3.9.5'),
```

```suggestion
    ('Python', '3.9.5', '-bare'),
```

We should use the less heavy `-bare` version of Python as the build dependency.

---

Fixed!